validation-worker/.gitignore

```diff
@@ -0,0 +1,329 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
It can be moved to the root of the repo.

Reply:
Deleted, already a version at root.